instrumentation/glog/CMakeLists.txt

```
@@ -69,6 +69,8 @@ if(OPENTELEMETRY_INSTALL)
 endif() # OPENTELEMETRY_INSTALL
 
 if(BUILD_TESTING)
```

To make this available for ctest in CI

```suggestion
if(BUILD_TESTING)
  enable_testing()
```

---

Thanks, fixed.